.env.development

```diff
@@ -39,3 +39,5 @@ FEATURE_FILE_ATTACHMENT=true
 IS_MAINTENANCE_ALERT_ENABLED=''
 MAINTENANCE_ALERT_MESSAGE=''
 MAINTENANCE_ALERT_START_TIMESTAMP=''
+TABLEAU_URL='https://enterprise-tableau.edx.org'
```

Comment:
shouldn't this value come from remote config?

Reply:
The one from the remote config will overwrite this value.

Reply:
No